src/ProductConstructionService/ProductConstructionService.Api/Queue/JobProcessorScopeManager.cs

```diff
@@ -10,15 +10,17 @@ public enum JobsProcessorState
     // The JobsProcessor isn't doing anything
     Stopped,
     // The JobsProcessor will finish its current job and stop
-    Stopping
+    Stopping,
+    // The JobsProcessor is waiting for VMR cloning to finish
+    WaitingForVmrClone
 }
 
-public class JobsProcessorScopeManager
+public class JobProcessorScopeManager
 {
-    public JobsProcessorScopeManager(bool workOnStartup, IServiceProvider serviceProvider)
+    public JobProcessorScopeManager(bool waitingForVmrCloneOnStartup, IServiceProvider serviceProvider)
```

Comment:
Same as above, remove the specifics and make it a generic wait for startup initialization